upgrade/helpers/tasks.py

```diff
@@ -923,3 +923,54 @@ def repository_cleanup(repo_name):
     for fname in os.listdir('/etc/yum.repos.d/'):
         if repo_name in fname.lower():
             os.remove('/etc/yum.repos.d/{}'.format(fname))
+
+
+def update_scap_content():
+    """ The purpose of this function is to perform deletion of old scap-contents
+        and then uploading new scap-contents. It also updates scap-policies and assign
+         new scap-content to them """
+
+    def scap(content_type, content_name):
+        for i in range(len(content_type)):
```

Comment:
The function is too lengthy, please split it into 2 to 3 part to reduce complexity and make it more understandable.

Reply:
Agreed, what @devendra104 said, this is way to complicate for updating content and policies. 
If necessary , just delete all policies and re-create them

Reply:
Use explicit varaiable names , rather than abbrevations

Reply:
@san7ket @devendra104 @ntkathole updated PR.